README.md

```diff
@@ -88,6 +90,14 @@ $ curl -H "Authorization: Bearer $TOKEN" -H 'Content-Type: application/json' -X
 
 ## Deploy to appengine
 
+Generate `app.yaml`
+
+```
+$ erb erb -T - app/concurrent-batch-agent/app.yaml.erb > app/concurrent-batch-agent/app.yaml
```

Comment:
I think that 'erb' is an unnecessary argument.

Reply:
@y-hattori Thanks. I removed it.